style.css

```diff
@@ -0,0 +1,114 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+#bgimage {
+  position: absolute;
+  background-image: url("images/Illustration.png");
+  background-position: center;
+  background-repeat: no-repeat;
+  padding-left: 165%;
+  padding-top: 1000px;
+  z-index: 1;
+}
```

Comment:
- [ ] As mentioned by the previous reviewer kindly fix the white space on the right side of your page, you still have the white space.
![Screenshot from 2022-07-02 11-29-53](https://user-images.githubusercontent.com/74501523/176995246-cf695d15-3771-4a65-be3f-1a57215e4ea8.png)

- [ ] To fix it you need to your background image to your `.body` class not create a `div` just for your background image, and then you can arrange your image :upside_down_face: :upside_down_face: